packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx

```diff
@@ -173,7 +192,7 @@ const ScreenSmall = observer(({ is_trade_enabled }) => {
 
     const has_callputequal_duration = hasDurationForCallPutEqual(
         contract_types_list,
-        duration_unit,
+        duration_unit as 's' | 'd' | 'm' | 'h' | 't',
```

Comment:
```suggestion
        duration_unit,
```
please do the following change in packages/trader/src/Stores/Modules/Trading/Helpers/allow-equals.ts type instead:
<img width="1048" alt="Screenshot 2023-09-08 at 12 08 16 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/f3282411-bc11-490a-8136-5ea51995709d">

Reply:
Will do, thank you